src/vizier/services/agent/pem/pem_main.cc

```diff
@@ -41,6 +42,8 @@ using ::px::vizier::agent::DefaultDeathHandler;
 using ::px::vizier::agent::PEMManager;
 using ::px::vizier::agent::TerminationHandler;
 
+constexpr std::string_view kLinuxHeadersPath = "/lib/modules";
```

Comment:
A little worried that there's an implicit relationship between this location and code we have in `pixie/src/stirling/utils/linux_headers.cc`. I guess one could argue that it's really BCC that's deciding to standardize on that location.

A comment about this being connected to BCC's search path might be warranted.

Reply:
Refactored this to a shared variable in fe6ead97447e2d5ef34eb1f73ffc72026b0f9a6a